### PR TITLE
new icon: spack (original, plain)

### PR DESCRIPTION
**Pull request by dennisklein**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date

* https://spack.io/
* https://github.com/spack/spack
* https://github.com/spack/spack.io/issues/55

### CODE REVIEW

File: icons/spack/spack-original.svg
Comment (Gautam25Raj) [inline icons/spack/spack-original.svg:1]: [SVG Standards](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards)

Use this instead:
```
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path fill="#0f3a80" d="m73.192 4.603 50.205 50.205a12.972 12.972 0 0 1 0 18.384l-50.205 50.205a12.972 12.972 0 0 1-18.384 0L4.603 73.192a12.972 12.972 0 0 1 0-18.384L54.808 4.603a12.972 12.972 0 0 1 18.384 0z"/><path fill="#ffa600" d="m48.59 32.389-8.336 9.261-2.602-2.343-5.466 13.918 13.27-6.891-2.602-2.344 8.332-9.254a13.126 13.126 0 0 1-2.596-2.347Zm8.16 4.478v.172l.135-.15c-.046-.006-.09-.016-.135-.022zm3.5 4.102-3.5 3.886v32.97h-3.5l5.25 14 5.25-14h-3.5zM58.5 14c-5.54 0-10 4.46-10 10s4.46 10 10 10c.356 0 .704-.03 1.049-.068l.719-.8c.164.167.335.325.507.481-2.62-2.38-4.275-5.807-4.275-9.613 0-3.853 1.692-7.321 4.37-9.705A9.949 9.949 0 0 0 58.5 14ZM24 54c-5.54 0-10 4.46-10 10s4.46 10 10 10 10-4.46 10-10-4.46-10-10-10zm10.33 17.855a13.113 13.113 0 0 1-2.475 2.475l10.936 10.936-2.475 2.474 13.612 6.188-6.188-13.612-2.474 2.475zM64 94c-5.54 0-10 4.46-10 10s4.46 10 10 10 10-4.46 10-10-4.46-10-10-10z"/><path fill="#fff" d="M104 54c-5.54 0-10 4.46-10 10s4.46 10 10 10 10-4.46 10-10-4.46-10-10-10zM67.75 36.867v40.957h-3.5l5.25 14 5.25-14h-3.5V36.867A12.96 12.96 0 0 1 69.5 37a12.95 12.95 0 0 1-1.75-.133Zm25.92 34.988L82.734 82.791l-2.474-2.475-6.188 13.612 13.612-6.188-2.475-2.474L96.145 74.33a13.113 13.113 0 0 1-2.475-2.475ZM79.41 32.389a13.127 13.127 0 0 1-2.597 2.35l8.333 9.253-2.601 2.342 13.27 6.89-5.467-13.917-2.602 2.343zm-19.142.744-16.3 18.103-2.6-2.341L35.9 62.81l13.27-6.891-2.602-2.342L63.055 35.27a13.09 13.09 0 0 1-2.787-2.138ZM69.5 14c-5.54 0-10 4.46-10 10s4.46 10 10 10 10-4.46 10-10-4.46-10-10-10z"/></svg>
```

File: icons/spack/spack-plain.svg
Comment (Gautam25Raj) [inline icons/spack/spack-plain.svg:1]: Use this instead:
```
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="M64 .784a12.96 12.96 0 0 0-9.192 3.82L4.604 54.808a12.972 12.972 0 0 0 0 18.385l50.204 50.204a12.972 12.972 0 0 0 18.384 0l50.204-50.204a12.972 12.972 0 0 0 0-18.385L73.192 4.604A12.96 12.96 0 0 0 64 .784ZM58.5 14a10.207 10.207 0 0 1 2.02.203c.126.025.25.055.374.086C58.207 16.673 56.5 20.139 56.5 24c0 3.731 1.603 7.083 4.134 9.456a13.156 13.156 0 0 0 2.419 1.815L46.568 53.579l2.6 2.341L35.9 62.812l5.466-13.917 2.602 2.342L56.75 37.04v-.171l.14.015 2.65-2.943c-.342.036-.688.06-1.04.06a10.11 10.11 0 0 1-2.861-.419 9.928 9.928 0 0 1-.92-.322c-.068-.027-.136-.054-.202-.083a9.927 9.927 0 0 1-.7-.336l-.145-.076a9.989 9.989 0 0 1-3.288-2.924c-.074-.102-.15-.201-.22-.306a9.962 9.962 0 0 1-.407-.672l-.036-.06c-.048-.087-.087-.179-.132-.267a9.904 9.904 0 0 1-.26-.542c-.05-.115-.094-.234-.14-.351a9.913 9.913 0 0 1-.181-.5 9.94 9.94 0 0 1-.25-.903 9.95 9.95 0 0 1-.078-.367 10.057 10.057 0 0 1-.094-.625c-.012-.096-.028-.192-.037-.29-.03-.315-.049-.634-.049-.958a10.207 10.207 0 0 1 .202-2.02 9.965 9.965 0 0 1 5.024-6.776 9.929 9.929 0 0 1 3.749-1.152A10.21 10.21 0 0 1 58.5 14zm11 0c5.54 0 10 4.46 10 10 0 5.541-4.46 10-10 10a9.969 9.969 0 0 1-6.466-2.366l-.03-.026a10 10 0 0 1-.783-.751c-.051-.054-.1-.11-.15-.165a9.983 9.983 0 0 1-.589-.718c-.049-.066-.1-.13-.147-.198a9.95 9.95 0 0 1-.595-.94c-.002-.005-.006-.01-.009-.014a9.929 9.929 0 0 1-.492-1.04c-.035-.088-.063-.178-.096-.267a9.916 9.916 0 0 1-.359-1.175 9.995 9.995 0 0 1-.156-.807c-.018-.119-.039-.235-.053-.355A10.152 10.152 0 0 1 59.5 24c0-.399.03-.79.075-1.177.014-.119.035-.235.053-.352a9.993 9.993 0 0 1 .247-1.167c.08-.285.171-.563.274-.837.03-.08.057-.163.09-.243.146-.36.309-.71.494-1.048a9.95 9.95 0 0 1 .604-.954c.046-.065.096-.128.143-.192.187-.251.385-.493.594-.725.049-.054.097-.108.147-.161a10 10 0 0 1 .812-.776A9.975 9.975 0 0 1 69.5 14zM48.59 32.39a13.12 13.12 0 0 0 2.597 2.347l-8.333 9.255 2.602 2.341-13.27 6.892 5.466-13.917 2.602 2.342zm30.82 0 8.336 9.26 2.602-2.342 5.465 13.917-13.27-6.892 2.603-2.341-8.334-9.255a13.12 13.12 0 0 0 2.597-2.347zm-11.66 4.48c.574.078 1.155.131 1.75.131s1.176-.053 1.75-.131v40.957h3.5l-5.25 14-5.25-14h3.5zm-7.5 4.1v36.857h3.5l-5.25 14-5.25-14h3.5v-32.97ZM24 54c5.54 0 10 4.46 10 10 0 5.541-4.46 10-10 10s-10-4.459-10-10c0-5.54 4.46-10 10-10zm80 0c5.54 0 10 4.46 10 10 0 5.541-4.46 10-10 10a9.97 9.97 0 0 1-8.204-4.27A9.97 9.97 0 0 1 94 64c0-5.54 4.46-10 10-10zM34.339 71.865 45.266 82.79l2.475-2.474 6.188 13.612-13.613-6.187 2.475-2.476L31.864 74.34a13.13 13.13 0 0 0 2.475-2.474zm59.322 0a13.13 13.13 0 0 0 2.475 2.474L85.209 85.266l2.475 2.476-13.613 6.187 6.188-13.612 2.475 2.474zM64 94c5.54 0 10 4.46 10 10 0 5.541-4.46 10-10 10s-10-4.459-10-10c0-5.54 4.46-10 10-10z"/></svg>
```

**Review by Gautam25Raj — CHANGES_REQUESTED** [review]

File: icons/spack/spack-original.svg
Comment (dennisklein) [reply icons/spack/spack-original.svg:1]: Done, thx!

File: icons/spack/spack-plain.svg
Comment (dennisklein) [reply icons/spack/spack-plain.svg:1]: Done, thx!

File: devicon.json [hunk 1]
```diff
@@ -8861,6 +8861,26 @@
             }
         ]
     },
+    {
+        "name": "spack",
+        "altnames": [],
+        "tags": [
+            "package-manager",
+            "package",
+            "manager"
```
Comment (Panquesito7) [inline devicon.json:8870]:
```suggestion
            "manager",
            "python",
            "open-source"
```

Comment (dennisklein) [reply devicon.json]: Done :)

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀

image

image

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thank you for a great contribution! 🚀

**Pull request merged**